Source/CarthageKit/Project.swift

```diff
@@ -371,6 +371,36 @@ public final class Project { // swiftlint:disable:this type_body_length
 					}
 					.map { $0.0.name }
 					.collect()
+		}
+	}
+
+	/// Finds all the transitive dependencies for the current project, grouped by their parent dependency.
```

Comment:
Reading this and looking at the return value, I'm not sure which `Dependency` is which. I think it deserves some clarification.